File: aspnet-core/MB.STA/MB.STA.Entities/Car.cs
using MB.STA.Utils.Enums;
using System;

namespace MB.STA.Entities
{
    public class Car
    {
        public int Id { get; set; }
        public string PlateNumber { get; set; }
        public string Name { get; set; }
        public DateTime MakeYear { get; set; }
        public FuelType FuelType { get; set; }
        public CarType CarType { get; set; }

        public int? DriverId { get; set; }
        public Driver Driver { get; set; }
    }
}

File: angular/src/app/shared/models/Car.ts
import { CarType } from "../enums/CarType.enum";
import { FuelType } from "../enums/FuelType.enum";

export interface Car {
    id: number;
    plateNumber: string;
    name: string;
    model: string;
    makeYear: string;
    fuelType: FuelType;
    carType: CarType;
}